source/tools/themefy.gd

```diff
@@ -7,61 +7,46 @@ func _run():
 	var interface = get_editor_interface()
 	var filesys = interface.get_resource_filesystem()
 	var dir = filesys.get_filesystem_path(PATH)
-	
+	var count = 0
 	for f in dir.get_file_count():
 		var path = dir.get_file_path(f)
 		if not filesys.get_file_type(path) == "Theme":
 			continue
 		var theme = load(path)
+		print("Creating theme for ", path)
 		theme.set_default_font(load("res://interface/fonts/kenvector_future_14.tres"))
 		set_stylebox(theme)
 		set_color(theme)
 		var subdir_index = dir.get_parent().find_dir_index(dir.get_file(f).replace("_theme.tres", ""))
 		var subdir = dir.get_parent().get_subdir(subdir_index)
 		set_icons(theme, subdir)
 		ResourceSaver.save(theme.get_path(), theme)
+		count +=1
 	
 	filesys.scan()
+	print("Correctly created ", count, " themes")
 	
 func set_stylebox(theme):
 	for t in theme.get_type_list(""):
 		for s in theme.get_stylebox_list(t):
 			if t == "Tree":
 				continue
-			elif t == "Label":
-				var path = theme.get_path().replace("_theme.tres", "/") + "empty.tres"
-				var box = load(path)
-				theme.set_stylebox(s, t, box)
-				continue
-			elif "check" in t.to_lower():
-				var path = theme.get_path().replace("_theme.tres", "/") + "empty.tres"
-				var box = load(path)
-				theme.set_stylebox(s, t, box)
-				print("check")
-				continue
-			if s == "grabber_area":
-				var path = theme.get_path().replace("_theme.tres", "/") + "scroll_focus.tres"
-				var box = load(path)
-				theme.set_stylebox(s, t, box)
-				continue
-			elif s == "slider":
-				var path = theme.get_path().replace("_theme.tres", "/") + "scroll.tres"
+			var path = theme.get_path().replace("_theme.tres", "/")
+			if t == "Label" or t == "check":
+				path += "empty.tres"
 				var box = load(path)
-				theme.set_stylebox(s, t, box)
-				continue
-			elif s == "bg":
-				var path = theme.get_path().replace("_theme.tres", "/") + "scroll.tres"
-				var box = load(path)
-				theme.set_stylebox(s, t, box)
-				continue
-			elif s == "fg":
-				var path = theme.get_path().replace("_theme.tres", "/") + "scroll_focus.tres"
-				var box = load(path)
-				theme.set_stylebox(s, t, box)
-				continue
-			var path = theme.get_path().replace("_theme.tres", "/") + s + ".tres"
+			if s == "grabber_area" or s == "fg":
+				path += "scroll_focus.tres"
+			elif s == "slider" or s == "bg":
+				path += "scroll.tres"
+			else:
+				var file = File.new()
+				path += s + ".tres"
+				# check if the path exists, first
+				if not file.file_exists(path):
+					continue
 			var box = load(path)
-			theme.set_stylebox(s, t, box)
+			theme.set_stylebox(s,t,box)
```

Comment:
Keep the spaces after commas `,`. Check out [Godot's GDScript guidelines](http://docs.godotengine.org/en/latest/getting_started/scripting/gdscript/gdscript_styleguide.html#whitespace) for more on that! :pig: